Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+name = "pypi"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+"boto3" = "*"
```

Comment:
Is there any reason not to use the same constraints as requirements.txt?

Reply:
I do not understand. I generated lockfile based on requirements.txt, but somehow I have double quotation.

Reply:
Oh, I meant the versioning.  But now looking at the docs it seems like pipenv's recommendation is to unpin things here are rely on the lockfile.